code/unclassified/src/unique_number/unique_num_stl.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+#include <vector>
+
+#include <algorithm>
+
+using namespace std;
```

Comment:
Same as before.

Reply:
can you help we to understand what is clang format?

Reply:
@Ankit152 Use [this website](https://zed0.co.uk/clang-format-configurator/) paste your code in, set indent width to 4 spaces, and reformat your code.